PWGLF/Tasks/cascpostprocessing.cxx

```diff
@@ -106,7 +106,7 @@ struct cascpostprocessing {
     TString CutLabel[26] = {"All", "MassWin", "y", "EtaDau", "DCADauToPV", "CascCosPA", "V0CosPA", "DCACascDau", "DCAV0Dau", "rCasc", "rCascMax", "rV0", "rV0Max", "DCAV0ToPV", "LambdaMass", "TPCPr", "TPCPi", "TOFPr", "TOFPi", "TPCBach", "TOFBach", "ctau", "CompDecayMass", "Bach-baryon", "NTPCrows", "OOBRej"};
     TString CutLabelSummary[29] = {"MassWin", "y", "EtaDau", "dcapostopv", "dcanegtopv", "dcabachtopv", "CascCosPA", "V0CosPA", "DCACascDau", "DCAV0Dau", "rCasc", "rV0", "DCAV0ToPV", "LambdaMass", "TPCPr", "TPCPi", "TOFPr", "TOFPi", "TPCBach", "TOFBach", "proplifetime", "rejcomp", "ptthrtof", "bachBaryonCosPA", "bachBaryonDCAxyToPV", "NTPCrows", "OOBRej", "rCascMax", "rV0Max"};
 
-    registry.add("hCandidate", "hCandidate", HistType::kTH1F, {{25, -0.5, 24.5}});
```

Comment:
That's related to another issue that we had in the postprocessing task: "Warning in <TH1Merger::CheckForDuplicateLabels>: Histogram hCandidate has duplicate labels in the x axis. Bin contents will be merged in a single bin"
(https://alimonitor.cern.ch/train-workdir/tests/0010/00106822/stdout.log)
I think that was happening because we had 25 bins in hCandidate histo, but there're only 24 labels, so the label for the 25th bin was taken from CutLabel[24] (line 111), which caused an undefined behavior.

Reply:
I just noticed that I had outdated version of postprocessing, so there should be 26 bins.

Reply:
bingo